### PR TITLE
Multitools can change the I/O dirs of ORMs

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: You can now change the input/output directons for Ore Redemption Machines by using a multitool on them with the panel open.
/:cl:

Removed get_(counter)clockwise_dir procs as they were pointless, we have turn().

### CODE REVIEW

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -923,13 +923,19 @@ var/list/WALLITEMS_INVERSE = typecacheof(list(
 
 //Get the dir to the RIGHT of dir if they were on a clock
 //NORTH --> NORTHEAST
-/proc/get_clockwise_dir(dir)
-	. = angle2dir(dir2angle(dir)+45)
+/proc/get_clockwise_dir(dir, cardinal = FALSE)
```
Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: firstly, no.
secondly use ```turn()``` where you need to.

I only made these clockwise procs as shortcuts to +45, they should also really be macros so now that's your job
```#define GET_CLOCKWISE_DIR(dir) turn(dir, 45)```, please update their calls accordingly to.

Comment (XDTM) [inline code/__HELPERS/unsorted.dm]: sorry about that, i didn't notice we had a turn proc

Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: nobody does!

**Comment by XDTM** [comment]
Turns out those procs were only used on a single proc right below them, i just replaced them with the turn proc.

**Comment by RemieRichards** [comment]
that's fine too.

**Comment by RemieRichards** [comment]
Why did you invert them...

**Comment by XDTM** [comment]
2:07:32 PM <+GinjaNinja32> XDTM, turn(dir, 45) is *counter*-clockwise, not clockwise 
2:07:42 PM <+GinjaNinja32> !dm turn(NORTH, 45) == NORTHWEST

**Comment by RemieRichards** [comment]
that's retarded.

**Comment by XDTM** [comment]
tell that to byond i guess

**Comment by ExcessiveUseOfCobblestone** [comment]
glad to see someone did this

**Pull request merged**